.github/workflows/build-and-lint-front.yml

```diff
@@ -20,4 +20,4 @@ jobs:
       - working-directory: types
         run: npm install && npm run build
       - working-directory: front
-        run: npm install && npm run tsc && npm run lint && npm run format:check
+        run: npm install && npm run tsc && npm run docs && npm run lint && npm run format:check
```

Comment:
do we absolutely need to add it here?

Reply:
I'd say yes because docs can be breakable as well and we want to know if a PR will break the docs, no?

Reply:
Actually I moved it to just a check like we do with the format check.